### DISCUSSION TITLE
MSSQL_CONNECT Die?

### DISCUSSION THREAD

**Discussion by kadugi** [FreeTDS/freetds#613] [General]
1 day ago all my sites using mssql_connect stopped working :(  anyone knows whats the problem? Without any change by my side or update.  Changed to sqlsrv and worked.

**Comment by freddy77** [comment]
Could you share some more details? I suppose you are talking about PHP functions seeing `mssql_connect`. Which versions of various software? Any firewall/network changes? Any system changes? Cloud? Which cloud?

**Reply by kadugi** [reply]
Yes, its about PHP functions ; 

Teste with php 5.3 ; php 5.6 ; Tested at least with 6 differents shared accounts using cpanel ; + 1 VPS using Ubuntu 22.02 + aapanel . All with freetds.

Nothing has been changed, it just stopeed work without any kind of log. 

Check Socket to port 1433  and its ok, connection 100% ; But MSSQL connection Receive Unknown error and cant connect. It only worked again when I migrated to sqlsrv;
